cmd/seid/cmd/debug.go

```diff
@@ -217,11 +219,20 @@ func ReadTree(db dbm.DB, version int, prefix []byte) (*iavl.MutableTree, error)
 	return tree, err
 }
 
-func PrintKeys(tree *iavl.MutableTree) []byte {
+func PrintKeys(tree *iavl.MutableTree, moduleParser ModuleParser) []byte {
 	fmt.Println("Printing all keys with hashed values (to detect diff)")
 	lines := []byte{}
 	tree.Iterate(func(key []byte, value []byte) bool { //nolint:errcheck
 		printKey := parseWeaveKey(key)
+		// parse key if we have a parser
+		if moduleParser != nil {
+			parsed, err := moduleParser(key)
+			if err != nil {
+				printKey = strings.Join([]string{printKey, err.Error()}, " | ")
+			} else {
+				printKey = strings.Join(append([]string{printKey}, parsed...), " | ")
+			}
+		}
 		digest := sha256.Sum256(value)
```

Comment:
Are we able to get the values decoded too?

Reply:
Sometimes the same acc would have different balances that could help

Reply:
we could potentially add decoded values too, but that would likely be a decent amount more work since we would have to deserialize protos for all the stored types, and then output them to the file. I'd prefer that we first make the keys more human readable for other modules, and then we can look at maybe parsing values, since even now, we should be able to perform queries via `seid` for module specific data for a given height once we know where the data discrepancy exists.

Reply:
wdyt @BrandonWeng